src/main/java/uk/gov/hmcts/cmc/claimstore/services/search/CCDCaseRepository.java

```diff
@@ -77,4 +77,14 @@ private void logClaimDetails(List<Claim> dbClaims, List<Claim> ccdClaims) {
 
         return claim;
     }
+
+    @Override
+    public Optional<Claim> linkDefendant(String externalId, String defendantId, String authorisation) {
+        Optional<Claim> claim = legacyCaseRepository.getClaimByExternalId(externalId);
```

Comment:
That does not look right that CCD implementation uses DB for linking. CCD repository should be used instead here.

Reply:
this is step 1

Reply:
will be a follow up PR to do that

Reply:
In that case please throw `NotImplemenetdException` and disable CCD in dev until fully functional. Providing two implementations in the same way as we have done it for few retrieval endpoints (retrieve both and log) is just unnecessary effort.

Reply:
This keeps it functional for now though? 
I'll remove it in a follow up but QA will want CCD enabled for now